### PR TITLE
In memory longevity ratio

### PR DESCRIPTION
**Pull request by roydahan**

### CODE REVIEW

File: test-cases/longevity/longevity-in-memory-36GB-4days.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+stress_cmd:        ["cassandra-stress mixed ratio(write=1,read=8) cl=QUORUM duration=5760m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..21000000 -col 'size=FIXED(200) n=FIXED(5)' -log interval=10"]
+stress_read_cmd:   ["cassandra-stress read                        cl=QUORUM duration=5760m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -pop seq=1..21000000 -col 'size=FIXED(200) n=FIXED(5)' -log interval=10",
+                    "cassandra-stress counter_read                cl=QUORUM duration=5760m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=10  -pop seq=1..12345678"]
+run_fullscan: 'random, 5' # 'ks.cf|random, interval(min)''
+n_db_nodes: 5
+n_loaders: 2
+n_monitor_nodes: 1
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+user_prefix: 'longevity-36gb-in-memory-VERSION'
```
Comment (fruch) [inline test-cases/longevity/longevity-in-memory-36GB-4days.yaml]: you should remove VERSION from here...

**Comment by fruch** [comment]
you pissed one of the precommit hooks:
https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-1020/1/execution/node/20/log/

**Comment by amoskong** [comment]
We need to backport this to branch-2019.1

**Comment by amoskong** [comment]
image

**Comment by amoskong** [comment]
@roydahan This PR introduced a new issue, let's fix it before backport.
The ratio parameter should be wrapped by single quotes.

**Pull request merged**